Makefile

```diff
@@ -34,9 +34,9 @@ build: ## Builds the docker-compose environment
 run: build ## Builds & runs the docker environment
 	@docker-compose $(docker_files) up --force-recreate
 migrate:
-	@NODE_ENV=development knex migrate:latest --cwd ../api/src/sql/ --migrations-directory ./migrations
+	@NODE_ENV=development AWS_ACCOUNT_ID=000000000000 AWS_REGION=eu-west-1 knex migrate:latest --cwd ../api/src/sql/ --migrations-directory ./migrations
```

Comment:
Nitpick: Could this be just one 0 maybe? Or a string that says "mockAccountId" or something?

Reply:
Hmm... we use 000000000000 for the AWS_ACCOUNT_ID for development, so I think it would be good if we use that here too?